src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import { Header, Home, Footer } from './components';
+
+function App() {
+  const { data, isLoading } = useFeaturedBanners();
+  console.log(data, isLoading);
+
+  return (
+    <>
+      <Header />
+      <Home />
+      <Footer text='Ecommerce created during Wizeline’s Academy React Bootcamp' />
```

Comment:
in this case, the footer is going to be re-used in another section? if the awnser is yer...ij ok to use the text as a prop....if not, maybe the text should be part of the foot state.